packages/web-components/src/components/footer/__stories__/README.stories.mdx

```diff
@@ -146,5 +146,6 @@ See [our README](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/t
 | `<dds-footer-logo>`      | `data-autoid="dds--footer-logo"`       | Component   |
 | `<dds-locale-button>`    | `data-autoid="dds--footer-locale-btn"` | Interactive |
 | `<dds-legal-nav>`        | `data-autoid="dds--legal-nav"`         | Component   |
+| `<dds-privacy-cp>`       | `data-autoid="dds--privacy-cp"`        | Component   |
```

Comment:
To reflect the actual tag name in use:

```suggestion
| `<dds-legal-nav-cookie-preferences-placeholder>` | `data-autoid="dds--privacy-cp"`        | Component   |
```

Reply:
@asudoh Oops yes, you are right. Thanks!